src/calculate-deal.mo

```diff
@@ -0,0 +1,79 @@
+import Iter "mo:base/Iter";
+import Nat "mo:base/Nat";
+
+module {
+
+  public func calculateDeal<Order>(
```

Comment:
Maybe we can call this function `matchOrders`.

Reply:
renamed